aten/src/ATen/native/native_functions.yaml

```diff
@@ -829,6 +829,11 @@
 
 - func: attn(Tensor q, Tensor k, Tensor v) -> (Tensor, Tensor)
   variants: function
+  dispatch:
+    CompositeExplicitAutograd: attn
+
```

Comment:
This was confusing to me at first I thought that I would write: `CompositeExplicitAutograd: attn_backward` causing I am pointing to the backward kernel I wrote in FuncsManual. I was getting linker errors so I thought added the one below would create the symbol I wanted. Ultimately figured it out